### ISSUE TITLE
Adding new dataset into previously integrated datasets

### ISSUE THREAD

**Issue by smgroves** [brianhie/scanorama#70]
I am wondering about the feasibility of using scanorama (R version) in the face of new datasets. For example, I had 8 datasets that I wanted to integrate, and used scanorama's correct method to do so (let's call the integrated dataset X). The method seems to have done a nice job at integrating and correcting the datasets, and I would now like to use X as a reference for any new data to be integrated into the same space. My understanding of the method is that if I try to integrate X and a new dataset, both will be "corrected." Also, I'm not sure if it would be a bad idea to try to integrate X, which has already been corrected with scanorama, and a new dataset that has not been normalized or corrected in the same way. Would you suggest that I try to use the correct() method on X and the new dataset, and therefore X will have been corrected twice? 

I also thought about using scanorama as a method to integrate the reference space itself, as I have done, and then use a different integration method that does take a reference to place new datasets within the space of X without changing X itself. Any thoughts?

**Comment by brianhie** [comment]
Hi @smgroves,

Glad to hear integration seemed to have worked on the 8 datasets. You can do a few things. (1) You could just integrate the 9 datasets together with Scanorama to construct a new reference, but then maybe you'd have to redo any analysis on your reference X, which is why you're interested in not modifying X. But this is the most canonical way to use Scanorama. (2) As you described, integrating X and and 9th dataset seems fine, and there is an implementation detail that might be in your favor -- when considering two datasets, Scanorama will only modify the values in the smaller dataset in terms of # of cells, so if the 9th dataset is smaller than X, then you might not have to worry about modified values (but would be good to make sure this is the case). (3) The most time-consuming and involved direction is to try out some of the transfer learning for reference projection tools (e.g., https://github.com/theislab/scarches), where you finetune their pretrained model on your Scanorama-integrated reference X, then use the finetuned model for any further reference projections. But transfer learning is still highly experimental and I'm not sure how robust this would be/how much finetuning would be required.

Hope that helps somewhat! I'll close the issue since there's not an actionable bug for Scanorama, but I'm happy to answer additional questions on this thread.

**Comment by smgroves** [comment]
@brianhie thanks so much for your quick response! I think I'll try your suggestion #2 and see if the values in the first dataset change. Thanks!
